miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditSampleController.java

```diff
@@ -274,6 +277,8 @@ public ModelAndView setupForm(@PathVariable Long sampleId, ModelMap model) throw
         .map(Dtos::asDto)
         .collect(Collectors.toList()));
 
+    model.put("projects", mapper.writeValueAsString(projectService.list().stream().map(Dtos::asDto).collect(Collectors.toList())));
```

Comment:
Navel gazing question: should we limit to active projects?

Reply:
Yeah, I'll change it to only display active projects + current project if not active